assets/js/modules/analytics/util/account.test.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * Account helper tests.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { getAccountDefaults } from './account';
+
+describe( 'getAccountDefaults', () => {
+	const sufficientArgs = {
+		siteName: 'Example Site',
+		siteURL: 'https://example.com/',
+		timezone: 'Europe/Kiev',
+	};
+
+	const namelessArgs = {
+		siteURL: 'https://example.com/subsite-slug/',
+		timezone: 'Europe/Kiev',
+	};
```

Comment:
Just a minor suggestion/nitpick here but it would be a bit more clear in each test if the function either used the args inline (not DRY), or defined the valid/common pieces up here and then tests where something different is used, that arg would be provided in the test.

E.g. here in `describe` we could have some valid values defined for each using the same name as the key in the args:

```js
const siteName = 'Site Name';
const siteURL = 'https://example.com/';
const timezone = 'Europe/Kiev';
```

Then in tests, the args would be composed of these, or use a different value inline:

```js
expect( getAccountDefaults( { siteName, siteURL, timezone } ) )
// or for cases where one or more use a different value, provide it inline
expect( getAccountDefaults( { siteName, siteURL, timezone: 'UTC' } ) )
expect( getAccountDefaults( { siteName, siteURL: '', timezone } ) )
expect( getAccountDefaults( { siteName: '', siteURL, timezone } ) )
expect( getAccountDefaults( { siteName, siteURL } ) )
```

This is perhaps a bit of a personal preference, but it helps to keep the relevant information _in the test_ with a focus on what is different rather than needing to look back and fourth.

This also makes the named args variables unnecessary and avoids potential confusion that could come from those (i.e. "if there are `sufficientArgs`, are `namelessArgs` insufficient?")

Reply:
Updated.